hw/ip/keymgr/README.md

```diff
@@ -88,11 +88,7 @@ The key used for all 3 functions is the `CreatorRootKey`.
 
 The advancement from `CreatorRootKey` to the `OwnerIntermediateKey` is irreversible during the current power cycle.
 
-While in the CreatorRootKey state, the key from OTP is continuously captured and sensed.
-This provides some security benefit as the key is constantly background checked by the OTP.
-When an operation begins, the sampling is stopped.
-If at the conclusion of the operation the key manager stays in the same state, sampling begins again.
-If on the other hand key manager transitions to another state, OTP sampling is stopped until reset.
+Keymgr reads the root key from OTP in a single clock cycle. It assumes that when keymgr's internal FSM reaches to this clock cycle, OTP root key is already available (`valid` is set to 1). Otherwise, keymgr skips loading the root key.
```

Comment:
qq: what will happen if keymgr skips loading the root key?

Reply:
i wonder if we need a dv testcase for that in block or chip level?

Reply:
If keymgr skips loading the key (because `valid` is not asserted when FSM arrives to that stage), then invalid key is loaded:
https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/keymgr/rtl/keymgr_ctrl.sv;l=347-352

Then I think keymgr end ups in a sync error state. I found the error propagation of `keymgr` a bit too complicated to be sure but it seems to start here:
https://cs.opensource.google/opentitan/opentitan/+/master:hw/ip/keymgr/rtl/keymgr_input_checks.sv;l=95

Do you think that SVA would be sufficient here to check that this case never happens? Here is an issue to track this: #17521